app/views/conference_registrations/show.html.haml

```diff
@@ -89,29 +89,27 @@
           %span.fa-stack
             %i.fa.fa-square-o.fa-stack-2x
             %i.fa.fa-ticket.fa-stack-1x
-          Tickets
+          Ticket Purchases
```

Comment:
Shouldn't we move the heading also inside the ``tickets.any?``?
If there are no tickets, we don't need this section at all, do we?